### DISCUSSION TITLE
tbox v1.7.1

### DISCUSSION THREAD

**Discussion by waruqi** [tboox/tbox#198] [General]
### New features

* [#190](https://github.com/tboox/tbox/pull/190): Add fs watcher
* Add `tb_file_touch` api

### Changes

* Support wasm
* Support arm64 for windows
* Improve tb_file_info to detect symlink
* Improve tb_file_copy to support symlink
* Improve tb_directory_copy to support symlink

### 新特性

* [#190](https://github.com/tboox/tbox/pull/190): 添加文件系统状态监视器
* 添加 `tb_file_touch` 接口

### 改进

* 支持 wasm
* 支持 arm64 windows
* 改进 tb_file_info，支持判断符号链接
* 改进 tb_file_copy 支持符号链接
* 改进 tb_directory_copy 支持符号链接

<hr /><em>This discussion was created from the release <a href='https://github.com/tboox/tbox/releases/tag/v1.7.1'>tbox v1.7.1</a>.</em>

**Comment by Anivie** [comment]
为什么这么多年了文档还是那么空。。。

**Reply by waruqi** [reply]
没空
